### ISSUE TITLE
Prefer "Challenge Ratio" to "Ratio Challenge"

### ISSUE THREAD

**Issue by zepumph** [phetsims/ratio-and-proportion#199]
There are many spots in the description. @Matthew-Moore240 and @terracoda will confer before making the change.

**Comment by zepumph** [comment]
They conferred. We will change all usages and then review.

**Comment by zepumph** [comment]
Please review, especially "challenge ratios".

**Comment by Matthew-Moore240** [comment]
Looks to me like they were all changed. Closing

**Comment by terracoda** [comment]
I noticed in the opening paragraph, we have 2 mentions of "ratio":

> In the Play Area you have two movable hands and a set of **challenge ratios** to explore. Hands can be moved up and down individually or simultaneously to explore multiple places where the chosen **ratio challenge** is true. There are options to add plain or numbered tick marks shown as evenly-spaced horizontal lines that divide up the vertical space of the Play Area. The Control Area has a button to reset the sim.

@zepumph, please remove "ratio" from "ratio challenge" and just leave it as "challenge".

And then, based on our thinking and work in #196, please swap the words in the H3 heading that above  the combobox, so it reads/says, "Challenge Ratio."

**Comment by zepumph** [comment]
Please note the changes to both screen's summaries. Anything else here?

**Comment by terracoda** [comment]
@zepumph, sorry for my lack of clarity, I only wanted the second instance changed. The first instance should still say, "challenge ratios".

The h3 is now correct.

Just add back "challenge ratios" in the first sentence:
"In the Play Area you have two movable hands and a set of challenge ratios to explore."

**Comment by zepumph** [comment]
How's that? I also want to remind you to feel free to edit the json file when there are these simple changes to be made, either directly on github, https://github.com/phetsims/ratio-and-proportion/edit/master/ratio-and-proportion-strings_en.json, or in your local copy. Furthermore, in cases where you are just editing strings in the sim that we are working on, even if it broke a string pattern or something (unlikely), it would be quite low risk, and I can always double check if you would like.

I'm always happy to handle anything for you, but I don't want it to be more challenging for you to explain it to me than it would be for you to just do it.

**Comment by terracoda** [comment]
I figured, I could do it. I'll try next time.

**Comment by terracoda** [comment]
Looks good thank-you, @zepumph.
